both redp and nlog:
```
2024-04-10T06:02:02.1075946Z FAILED test_stream_smoke/test_query_smoke.py::test_run[two_level_global_aggr-3-subscribe-and-recover-two-level-global-aggr]
2024-04-10T06:02:02.1077589Z FAILED test_stream_smoke/test_query_smoke.py::test_run[changelog_stream13-103-global aggr on changelog_kv stream with single shard]
2024-04-10T06:02:02.1079219Z FAILED test_stream_smoke/test_query_smoke.py::test_run[changelog_stream13-107-global aggr on changelog_kv stream with single shard]
2024-04-10T06:02:02.1080734Z FAILED test_stream_smoke/test_query_smoke.py::test_run[query_state5-102-multiple_recover_global_aggr_in_substream]
2024-04-10T06:02:02.1081841Z ================== 4 failed, 839 passed in 1027.38s (0:17:07) ==================
```

```
2024-04-10T05:59:08.0823605Z =========================== short test summary info ============================
2024-04-10T05:59:08.0824853Z FAILED test_stream_smoke/test_query_smoke.py::test_run[two_level_global_aggr-3-subscribe-and-recover-two-level-global-aggr]
2024-04-10T05:59:08.0826472Z FAILED test_stream_smoke/test_query_smoke.py::test_run[changelog_stream13-103-global aggr on changelog_kv stream with single shard]
2024-04-10T05:59:08.0828115Z FAILED test_stream_smoke/test_query_smoke.py::test_run[changelog_stream13-107-global aggr on changelog_kv stream with single shard]
2024-04-10T05:59:08.0829619Z FAILED test_stream_smoke/test_query_smoke.py::test_run[query_state5-102-multiple_recover_global_aggr_in_substream]
2024-04-10T05:59:08.0830643Z ================== 4 failed, 846 passed in 858.35s (0:14:18) ===================
```

--- a/tests/stream/test_stream_smoke/0013_changelog_stream13.yaml
+++ b/tests/stream/test_stream_smoke/0013_changelog_stream13.yaml
@@ -24,6 +24,7 @@ tests:
       - single shard
       - checkpoint
       - emit changelog
+      - bug
     name: "global aggr on changelog_kv stream with single shard"
     description: sum_distinct and count_distinct for changelog_kv stream with single shard.
     steps:
@@ -275,6 +276,7 @@ tests:
       - changelog_kv
       - single shard
       - checkpoint
+      - bug
     name: "global aggr on changelog_kv stream with single shard"
     description: sum_distinct and count_distinct for changelog_kv stream with single shard.
     steps:

--- a/tests/stream/test_stream_smoke/0018_query_state5_substream.json
+++ b/tests/stream/test_stream_smoke/0018_query_state5_substream.json
@@ -85,7 +85,7 @@
         },
         {
             "id": 102,
-            "tags": ["query_state"],
+            "tags": ["query_state", "bug"],
             "name": "multiple_recover_global_aggr_in_substream",
             "description": "multiple recover global aggregation in substream from state checkpoint",
             "steps":[

--- a/tests/stream/test_stream_smoke/0030_two_level_global_aggr.yaml
+++ b/tests/stream/test_stream_smoke/0030_two_level_global_aggr.yaml
@@ -176,6 +176,7 @@ tests:
       - "emit changelog"
       - "changelog aggr"
       - "query_state"
+      - "bug"
     name: subscribe-and-recover-two-level-global-aggr
     description: subscribe to two level global aggregation query, then recover from checkpointed.
     steps:

